editor/src/clj/editor/keymap.clj

```diff
@@ -378,13 +381,14 @@
 ;; These are only (?) used in contexts where there is no text field
 ;; interested in the actual typable input.
 (def ^:private default-allowed-typable-shortcuts
-  #{"A"         ; :add
-    "E"         ; :rotate-tool
-    "F"         ; :frame-selection
-    "R"         ; :scale-tool
-    "W"         ; :move-tool
-    "Shift+A"   ; :add-secondary
-    "Shift+E"}) ; :erase-tool
+  #{"A"          ; :add
+    "E"          ; :rotate-tool
+    "F"          ; :frame-selection
+    "R"          ; :scale-tool
+    "W"          ; :move-tool
+    "Ctrl+Alt+Y" ; :async-reload
+    "Shift+A"    ; :add-secondary
+    "Shift+E"})  ; :erase-tool
```

Comment:
Mostly whitespace, but added "Ctrl+Alt+Y" here. It seems Ctrl+Alt+[Anything] is considered "typeable" under Windows, because the AltGr key is the same as pressing Ctrl+Alt. AltGr+Y appears to produce an `ü` on most keyboard layouts. I don't know, maybe we should choose a different shortcut?

Reply:
Maybe use <kbd>Ctrl+Y</kbd> on Windows?

Reply:
On second thought, I think I'll just revert this and change it to use `Shift+Cmd/Ctrl+Y` on all platforms.

Reply:
Sounds good!